sys/net/gnrc/transport_layer/tcp/gnrc_tcp_eventloop.c

```diff
@@ -248,9 +248,6 @@ static int _receive(gnrc_pktsnip_t *pkt)
     mutex_unlock(&list->lock);
 
     /* Call FSM with event RCVD_PKT if a fitting TCB was found */
-    /* cppcheck-suppress knownConditionTrueFalse
-     * (reason: tcb can be NULL at runtime)
-     */
     if (tcb != NULL) {
```

Comment:
Here the warning of `cppcheck` is correct: `tcb` must be `NULL`, since the `while (tcb)` has no `break` in it except when `MODULE_GNRC_IPV6` is defined (which `cppcheck` assumes to not be the case).